libopenarc/arc-util.h

```diff
@@ -6,6 +6,8 @@
 
 #include "build-config.h"
 
+#include <bsd/string.h>
```

Comment:
This will take some work in configure.ac to get right.  It doesn't exist on my FreeBSD development machine, for example, so we'll have to arrange to search for it.

Why is it needed here?  Isn't it only needed in arc-util.c, where the strlxxx() functions are needed?